ui/app/templates/vault/cluster.hbs

```diff
@@ -85,15 +85,26 @@
       {{#if flash.componentName}}
         {{component flash.componentName content=flash.content}}
       {{else}}
-        <h5 class="title is-5 has-text-{{if (eq flash.type 'warning') 'dark-yellow' flash.type}}">
-          {{get (message-types flash.type) "text"}}
-        </h5>
-        <span data-test-flash-message-body=true>
-          {{flash.message}}
-        </span>
-        <button type="button" class="delete" {{action close}}>
-          {{i-con excludeIconClass=true glyph="close" aria-label="Close"}}
-        </button>
+        <div class="message is-{{flash.type}}">
+          <div class="columns is-mobile is-variable is-1">
+            <div class="column is-narrow message-icon">
+              <ICon @glyph="{{get (message-types flash.type) "glyph"}}" @size="20" @excludeIconClass="true" />
```

Comment:
```suggestion
              <ICon @glyph="{{get (message-types flash.type) "glyph"}}" @size="20" @excludeIconClass={{true}} />
```
I think it'll work either way (since it's truthy), but with bools we normally want curlies around them.